great_expectations/datasource/generator/query_generator.py

```diff
@@ -101,7 +101,7 @@ def add_query(self, generator_asset, query):
 
     def get_available_data_asset_names(self):
         if self._queries_path:
-            defined_queries = [path for path in os.walk(self._queries_path) if str(path).endswith(".sql")]
+            defined_queries = [path for path in os.listdir(self._queries_path) if str(path).endswith(".sql")]
```

Comment:
```suggestion
            defined_queries = [path[:-4] for path in os.listdir(self._queries_path) if str(path).endswith(".sql")]
```

Reply:
Thanks for the suggestion @jcampbell . Fixed.